product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,86 @@
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of  Superset you will be logged out of EDB Cloud
+
+
+## Connecting Superset to your Cluster
+
+To connect superset to your cluster you need to provide a connection string for your cluster.
+As described [here](https://superset.apache.org/docs/databases/postgres#postgres) the connection string must have the following format:
+
+
+ `postgresql://{username}:{password}@{host}:{port}/{database}?sslmode=require`
+
+To find the host and port  of your cluster:
+
+
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+2. Go to the [Clusters](https://portal.edbcloud.com/clusters) page.
+3. Select the name of your cluster.
+4. Select the **Connect** tab and  find **Host** and **Port** .
+
+Connecting Superset to a EDB Cloud cluster is similar to connecting[ Superset to a new database](https://superset.apache.org/docs/creating-charts-dashboards/first-dashboard#connecting-to-a-new-database).
+To create a connection to an EDB Cloud cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+2. Navigate to **Analyze->Connections**.
+3. Select **+ Database **button on top right. This will open a dialog box.
+4. Enter **DATABASE NAME**.  Enter the connection string for your cluster as **SQLALCHEMY URI**, in the format described above.
+5. Check the connection using the **TEST CONNECTION** button and add it if you see success pop-up
+
+
+
+Upon successful connection, you are good to add datasets, charts and dashboards.
+
+
+!!! Note
+    Superset will only work with postgres clusters which are local to it or accessible on a public network. Add clusters from the region where the superset is deployed in the data plane.
+
+
+## Using  Superset Dashboards
+
+You can use Superset Dashboard to analyze data stored in your cluster. [This page](https://superset.apache.org/docs/creating-charts-dashboards/first-dashboard) contains instructions for creating a simple Superset dashboard.
+
+To view all available Superset Dashboards, navigate to** Analyze>Dashboards**
+
+
+## Using Superset Charts
+
+You can use Superset Charts to visualize data stored in your cluster. [Superset documentation ](https://superset.apache.org/docs/creating-charts-dashboards/exploring-data)contains instructions and examples of creating  Superset Chart.
+
+To view all available Superset Charts, navigate to **Analyze>Charts**.
+
+
+## Using  Superset SQL Lab
+
+You can use Superset SQL Lab to write queries to access/modify data stored in your cluster. To access SQL Lab, navigate to **Analyze>SQL Editor**.
+
+
+##  Superset Permissions
+
+Access to Superset data sources is controlled by Superset roles. Learn more about Superset Roles, Users and Permissions management in [Superset Security](https://superset.apache.org/docs/security).
+
+EDB Cloud account using the following special permission:
+
+
+
+* superset:admin
+* superset:alpha
```

Comment:
In auth0, when a user gets created we assign the role `owner:org_brCxzr08qr7RBEi1` to the user and this role has two Superset permissions `superset:admin` and `superset:alpha`.

My question is  - Can user have two superset permissions at a time?
@nstott
cc/ - @vishalsawale9

Reply:
I think - yes; it can happen as we only have 2 roles and in anyway `admin` will supersede `alpha`. But `alpha` cannot be perceived as `admin`.